src/storybook/pages/lean/lean.stories.ts

```diff
@@ -0,0 +1,156 @@
+import type { Meta, StoryObj } from '@storybook/web-components';
+import type { TemplateResult } from 'lit';
+import { html } from 'lit';
+
+import '../../../elements/alert.js';
+import '../../../elements/container.js';
+import '../../../elements/action-group.js';
+import '../../../elements/button.js';
+import '../../../elements/card.js';
+import '../../../elements/checkbox.js';
+import '../../../elements/form-field.js';
+import '../../../elements/header.js';
+import '../../../elements/icon.js';
+import '../../../elements/menu.js';
+import '../../../elements/paginator.js';
+import '../../../elements/radio-button.js';
+import '../../../elements/signet.js';
+import '../../../elements/table.js';
+
+import './lean.scss';
+
+const leanExampleTemplate = (): TemplateResult => html`
+  <div>
+    <sbb-header hide-on-scroll expanded>
+      <sbb-header-button icon-name="hamburger-menu-small" expand-from="small">
+        Menu
+      </sbb-header-button>
+      <div class="sbb-header-spacer"></div>
+      <sbb-header-link icon-name="magnifying-glass-small" href="/"> Search </sbb-header-link>
+      <sbb-header-button icon-name="user-small" class="sbb-header-shrinkable">
+        Sign in
+      </sbb-header-button>
+      <sbb-header-button icon-name="globe-small" id="language-menu-trigger" class="last-element">
+        English
+      </sbb-header-button>
+      <sbb-menu trigger="language-menu-trigger">
+        <sbb-menu-button aria-pressed="false">Deutsch</sbb-menu-button>
+        <sbb-menu-button aria-pressed="false">Français</sbb-menu-button>
+        <sbb-menu-button aria-pressed="false">Italiano</sbb-menu-button>
+        <sbb-menu-button icon-name="tick-small" aria-pressed="true"> English </sbb-menu-button>
+      </sbb-menu>
+      <a slot="logo" aria-label="Homepage" href="/">
+        <sbb-signet protective-room="panel"></sbb-signet>
+      </a>
+    </sbb-header>
+
+    <sbb-container expanded>
+      <section class="lean-section">
+        <div class="lean-container">
+          <sbb-form-field>
+            <label>Departure</label>
+            <sbb-select>
+              <sbb-option value="1" selected>Zug</sbb-option>
+              <sbb-option value="2">Lucerne</sbb-option>
+              <sbb-option value="3">Fribourg</sbb-option>
+            </sbb-select>
+          </sbb-form-field>
+          <sbb-form-field>
+            <label>Arrival</label>
+            <sbb-icon slot="prefix" name="magnifying-glass-small"></sbb-icon>
+            <input />
+            <sbb-icon slot="suffix" name="circle-question-mark-small"></sbb-icon>
+          </sbb-form-field>
+          <sbb-secondary-button>Clear departure</sbb-secondary-button>
+          <sbb-button>Search</sbb-button>
+        </div>
+      </section>
+      <section class="lean-section">
+        <div class="lean-container-half">
+          <sbb-table-wrapper>
+            <table class="sbb-table sbb-table-xs sbb-table--striped">
```

Comment:
I think the classes need to be adapted as well, so taking xs as standard in lean (@mcilurzo , really so small as standard?)

`styles/table.scss`

```scss
html.sbb-lean .sbb-table:not(.sbb-table-s, .sbb-table-m) {
  @include table.table--xs;
}
```

Reply:
answer of @mcilurzo: s should be the default in lean